.github/workflows/codeql.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.22'
+          go-version: '1.23'
```

Comment:
would be better to use `go-version-file: go.mod`?

Reply:
Would you like to push a PR with this change?
Yes, that it would be better if work out in this case

Reply:
yes, can work on that next week